charts/logging-pipeline-plumber/templates/deployment.yaml

```diff
@@ -34,6 +34,7 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           args: [
+            "-release-namespace={{ .Release.Name }}",
```

Comment:
can we rename this to aggregator service namespace.  It will usually match the release namespace but that is not the purpose of the variable,